new/LIT7235PPSolomonSmiths.xml

```diff
@@ -0,0 +1,77 @@
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient4943"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Magic"/>
+                    <term key="Prayers"/>
+                    <term key="Asmat"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-01-11">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dc:relation" active="LIT7235PPSolomonSmiths" passive="LIT7105NamesSolomon"><desc>Thematically 
+                        similar to <ref type="work" corresp="LIT7105NamesSolomon"/>.</desc></relation>
+                    <relation name="dc:relation" active="LIT7235PPSolomonSmiths" passive="LIT6497PPAynat"><desc>Thematically 
+                        similar to <ref type="work" corresp="LIT6497PPAynat"/>.</desc></relation>
+                    <relation name="dc:relation" active="LIT7235PPSolomonSmiths" passive="LIT6473PPAynaTela"><desc>Thematically 
+                        similar to <ref type="work" corresp="LIT6473PPAynaTela"/>.</desc></relation>
+                    <relation name="dc:relation" active="LIT7235PPSolomonSmiths" passive="LIT7234PPSolomonHand"><desc>Thematically 
+                        similar to <ref type="work" corresp="LIT7234PPSolomonHand"/>.</desc></relation>
+                </listRelation>
+            </div>
+            <div type="edition" subtype="incipit">
```

Comment:
here, on the contrary, you can specify the source for the incipit.

Reply:
It would also be useful to mention explicitly that the beginning of the text is missing, according to Strelcyn.